src/Conversations/HelloConversation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Telegram\Bot\Conversations;
+
+/**
+ * Class HelloConversation
+ * @package Telegram\Bot\Conversations
+ * @author Artiom Mocrenco <tjomamokrenko@gmail.com>
+ */
```

Comment:
Please remove package and author. You'll be credited in release notes. We need to be consistent.